### ISSUE TITLE
Add prefered screen orientation to profile settings

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/Osmand#5288]
For some navigation profiles it would be nice to have a fixed screen orientation. For example on a bicycle or motorcycle curvy roads and intersections can lead to the screen being rotated which is rather irritating and undesirable. A new option for each profile to choose between automatic (the default), horizontal and vertical would be nice.

There is of course a device-wide option to configure the screen orientation. However I usually forget to change it before starting to drive. Since the preferred orientation strongly depends on the specific navigation profile it would be nice to add such an option.

**Comment by thuryn** [comment]
I would like to have this feature too

**Comment by sonora** [comment]
You can configure this per profile, e.g. car, bicycle, ped, etc., see #7359, and it will then always of course also apply for any navigation using that profile.

**Comment by scaidermern** [comment]
> You can configure this per profile, e.g. car, bicycle, ped, etc., see #7359, and it will then always of course also apply for any navigation using that profile.

I linked these two issues by mistake. This one is about screen rotation (horizontal vs vertical) while the other one is about map rotation (compass / driving direction / north up).

**Comment by scaidermern** [comment]
Configuration of the screen orientation on a per-profile basis has been implemented in OsmAnd 3.5.0.
